src/indexer/index.ts

```diff
@@ -262,3 +268,43 @@ export async function getAssetInformation(
 		throw e;
 	}
 }
+
+/**
+ * Get parsed DAO information
+ * @param params object to decode
+ */
+export async function decodePSQLAppParams(params: any): Promise<DaoTableData> {
```

Comment:
```suggestion
export async function decodeAppParams(params: any): Promise<DaoTableData> {
```
PSQL is not related to app params. Some other name is needed.

Reply:
I have a function for `decodeAppParamsState` so I added PSQL to make it separate that the whole object received is decoded